searx/plugins/oa_doi_rewrite.py

```diff
@@ -40,9 +42,10 @@ def on_result(request, search, result):
 
     doi = extract_doi(result['parsed_url'])
     if doi and len(doi) < 50:
-        for suffix in ('/', '.pdf', '/full', '/meta', '/abstract'):
+        for suffix in ('/', '.pdf', '.xml', '/full', '/meta', '/abstract'):
             if doi.endswith(suffix):
                 doi = doi[:-len(suffix)]
+        logger.debug("found DOI: '%s'", doi)
```

Comment:
Is it a leftover? If not, please move the import to the same line as settings:

```python
from searx import logger, settings
```

Reply:
I decided to drop the logging (not really a need for).